packages/geospatial/src/processing/raster-vector-fusion.ts

```
@@ -0,0 +1,670 @@
+          longitude: feature.geometry.coordinates[0],
+          latitude: feature.geometry.coordinates[1],
+        });
+        break;
+      case 'MultiPoint':
+        for (const coord of feature.geometry.coordinates) {
+          points.push({ longitude: coord[0], latitude: coord[1] });
+        }
+        break;
+      case 'LineString':
+        // Sample along line
+        const line = turf.lineString(feature.geometry.coordinates);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)